docs/sources/administration/roles-and-permissions/access-control/plan-rbac-rollout-strategy/index.md

```diff
@@ -301,3 +305,41 @@ roles:
 ```
 
 - Or use [RBAC HTTP API]({{< relref "../../../../developers/http_api/access_control/#update-a-role" >}}).
+
+### Manage user permissions through teams
+
+In the scenario where you want users to have granted access by the team they belong to, we recommend to set users role to `No Basic Role` and let the team assignment assign the role instead.
+
+1. In Grafana, ensure the following configuration settings are enabled.
+   1. `users.auto_assign_org = true` this will automatically assign users to the default organization.
+   1. `users.auto_assign_org_id = <org_id>` this will automatically assign users to the specified organization.
+   1. `users.auto_assign_org_role = None` this will automatically assign users to the specified role.
+   1. Restart the Grafana instance.
+2. Create a team with the desired name.
+3. Set the fixed roles needed for the team.
+4. Add users to the team.
+
+This way, a user will be added automatically to the default organization and have no permissions until it has been assigned to a team.
+
+### Allow user to access only to a specific dashboard
```

Comment:
I don't think I agree with this scenario and would not mention it here, at least yet.
I strongly feel like "No Basic Role" is an incorrect answer to the problem: "Viewers can access all dashboards by default".
IMHO, this use case is actually a feature request to: "Allow modifying the default dashboard/folder permissions".
Wdyt?